app/components/shared/index.js

```diff
@@ -1,3 +1,6 @@
 export { default as CopyToClipboard } from "./CopyToClipboard";
 export { default as RouteTransition } from "./RouteTransition";
 export { default as Tooltip } from "./Tooltip";
+export { default as LinkToAccounts } from "./LinkToAccounts";
+
+export const Aux = ({ children }) => children;
```

Comment:
What is the benefit of this helper?

Reply:
It's a self immolating component.

This allows us to build components using a wrapper that doesn't appear in the final website markup.

I used it on the main page views so that they're placed directly into the markup without a wrapper "div".

This resolves styling issues without the need to create extra classnames to propagate parent layout styles (like height: 100%).

This is one of the new patterns from react 16. It's really powerful. :)